### ISSUE TITLE
REQUEST: Razer Blade 15 (2020) Base Model config

### ISSUE THREAD

**Issue by orfeaspgd** [hirschmann/nbfc#1060]
I have a Razer Blade 15 (2020) Base Model with RTX2060 i7 10th gen and the fans make a lot of noise even on idle. After looking up this issue I found out that a lot of users of this model have the same problem as I do. I would really appreciate if someone created a config about this machine. Thank you!

**Comment by wipeout12** [comment]
I have the same model, but with RTX2070. If you want to make it a bit quieter from power options set the maximum processor state to 99% it will disable CPU boost,

**Comment by orfpou** [comment]
I have already tried that, didn't change much, but I will try it again. I also turned off Turbo boost and from power options the cooling policy to passive (from active). Do you have any other suggestions?

**Comment by orfeaspgd** [comment]
> I have the same model, but with RTX2070. If you want to make it a bit quieter from power options set the maximum processor state to 99% it will disable CPU boost,

I have already tried that, didn't change much, but I will try it again. I also turned off Turbo boost and from power options the cooling policy to passive (from active). Do you have any other suggestions?

**Comment by wipeout12** [comment]
After disabling turbo my cpu doesn't go more than 25 watts. While playing league of legends and ~12 chrome tabs the laptop is almost silent, sometimes fans goes completely off for few seconds. However, you can lower your cpu wattage from throttle stop, but haven't tried it with disabled boost

**Comment by balintlaczko** [comment]
I don't think ThrottleStop or Intel XTU actually lets you undervolt the 10750H. Or has someone manage to do that?

BTW I tried to follow the (very nice!) documentation to find the read/write registers, and tried some test writes, but didn't seem to have any effect. But the methods listed are quite nice, if someone is willing to study the thing, I am sure it's possible to find them (I only spent around 2 hours on it).
